src/main.rs

```diff
@@ -442,18 +442,18 @@ fn process_environment_command(
                     "Environment '{}' not updated: same description",
                     env_name.unwrap()
                 ))?;
+            } else if parent_name.is_some() && parent_name.unwrap() != details.parent.as_str() {
+                error_message(format!(
+                    "Environment '{}' parent cannot be updated.",
+                    env_name.unwrap()
+                ))?;
+                process::exit(6);
```

Comment:
Is this like exit(42) (some arbitrary non-zero number)?

Reply:
Yes, just trying to give diversity in the value returned to maybe help isolate with less than full debug info.